tests/wpt/meta/css/css-flexbox/__dir__.ini

```diff
@@ -1 +1 @@
-prefs: ["layout.columns.enabled:true", "layout.flexbox.enabled:true"]
+prefs: ["layout.columns.enabled:true", "layout.flexbox.enabled:true"]
```

Comment:
It looks like this change is accidental?

Reply:
Think this one is probably a rebase conflict. Will undo. What I would like, to do soon (once this PR lands) is enable the `css-grid` tests, if that is something you'd be open to.